### DISCUSSION TITLE
How to theme KDE apps in Hyprland using pywal/wallust?

### DISCUSSION THREAD

**Discussion by iamhrigved** [hyprwm/Hyprland#7532] [Q&A]
I have all the env variables setup correctly and my Kvantum theme is [Edna](https://store.kde.org/p/1367055/). I researched a bit and found out that this is the directory structure of Kvantum config.
```
~/.config/Kvantum
├── kvantum.kvconfig
└── Edna
    ├── Edna.kvconfig
    └── Edna.svg
```
Then I changed the colors in the `Edna.kvconfig` file and it *mostly* worked. Many colors changed but some colors which required the svg file, didn't. So now, I changed the colors in the `Edna.svg` file by find-n-replace. Now, all of the colors changed and my dolphin was looking dope.

BUT

When I used to download something from Brave, the window which popped up wasn't my custom dolphin window. It had the theme [Otto](https://store.kde.org/p/1358260/) which was my KDE Plasma theme. I have no idea what is going on, please help me guys 🙏

And please note that my ultimate goal is to ditch Kvanutum completely and to be able to change KDE apps colors using a color generator like pywal or wallust. How can I implement this?

**Comment by jsah-mc** [comment]
Try this repo it maybe works for pywal but i will have to search for wallust [https://github.com/eylles/pywal16-libadwaita](url)
